gym/lib/gym/runner.rb

```diff
@@ -237,6 +239,76 @@ def copy_mac_app
       app_path
     end
 
+    def build_mac_pkg(app_path)
```

Comment:
Maybe this should get a better name that reflects "codesign and package"? 
`build` sounds pretty lame her - building happened before, this is finalizing it, right?